core/src/main/scala/kafka/server/KafkaApis.scala

```diff
@@ -116,6 +117,7 @@ class KafkaApis(val requestChannel: RequestChannel,
   val requestHelper = new RequestHandlerHelper(requestChannel, quotas, time)
   val aclApis = new AclApis(authHelper, authorizer, requestHelper, "broker", config)
   val configManager = new ConfigAdminManager(brokerId, config, configRepository)
+  val partitionRequestLimit = 2000
```

Comment:
Do we want a config to control this limit?

Reply:
Usually we need to be able to configure limits.

Reply:
Sure, added the config.